### PR TITLE
RED-36146 Support for Oracle Linux 7.6

### PR DESCRIPTION
**Pull request by Paz-Yanover**
@bmansheim - in addition, we need to check (probably with Alon, maybe Uri) whether Oracle Linux also "requires at least "Minimal Install" configuration _[- what does it mean????]_, OpenSSL 1.0.2, and firewall configuration." _[- maybe it's relevant only for the Oracle Linux 7 installations....?]_

### CODE REVIEW

**Comment by bmansheim** [comment]
@paz Please approve

File: content/rs/administering/designing-production/supported-platforms.md [hunk 1]
```diff
@@ -13,14 +13,15 @@ Make sure your system meets these requirements:
 - Only 64-bit operating systems are supported.
 - You must install Redis Enterprise Software directly on the host, not through system cloning.
 - You must install on a clean host with no other applications running so that all RAM is allocated to the OS and RS only.
+- Linux distributions must be installed with: "Minimal Install" configuration, OpenSSL 1.0.2, [firewall configuration]({{< relref "/rs/installing-upgrading/configuring/centos-rhel-7-firewall.md" >}})
```
Comment (Paz-Yanover) [inline content/rs/administering/designing-production/supported-platforms.md]: Ubuntu is also Linux distro and it doesn't require the comment of "Minimal Install" configuration, OpenSSL 1.0.2, and firewall configuration. It's probably relevant only to RHEL and Oracle Linux (- not really sure about the later)

**Comment by paz** [comment]
what

**Comment by Paz-Yanover** [comment]
Approved

**Pull request merged**